pkg/pquerier/select.go

```diff
@@ -104,8 +103,8 @@ func (queryCtx *selectQueryContext) start(parts []*partmgr.DBPartition, params *
 		queryCtx.totalColumns = queryCtx.frameList[0].Len()
 	}
 
-	frameIter := NewFrameIterator(queryCtx)
-	return frameIter, nil
+	frameIter, err := NewFrameIterator(queryCtx)
+	return frameIter, err
```

Comment:
I think you can actually just `return NewFrameIterator(queryCtx)` in this case 😄